setup.cfg

```diff
@@ -100,10 +100,23 @@ doctest_plus = enabled
 text_file_format = rst
 open_files_ignore = "astropy.log" "/etc/hosts" "*.ttf"
 remote_data_strict = true
-addopts = -p no:warnings --doctest-rst
+addopts = --doctest-rst
 asdf_schema_root = astropy/io/misc/asdf/data/schemas
 xfail_strict = true
 qt_no_exception_capture = 1
+filterwarnings =
+    error
+    ignore:unclosed file:ResourceWarning
+    ignore:unclosed <socket:ResourceWarning
+    ignore:unclosed <ssl.SSLSocket:ResourceWarning
+    ignore:The timefunc function is deprecated
+    ignore:numpy.ufunc size changed:RuntimeWarning
+    ignore:.*'np.bool_' scalars to be interpreted as an index:DeprecationWarning
```

Comment:
@astrofrog , is this the correct way to handle this warning that I see in https://travis-ci.org/astropy/astropy/jobs/612683779 coming out of visualization tests?

```
        left, right = self.xaxis.limit_range_for_scale(left, right)
>       left, right = sorted([left, right], reverse=reverse)
E       DeprecationWarning: In future, it will be an error for 'np.bool_' scalars to be interpreted as an index
.../python3.8/site-packages/matplotlib/axes/_base.py:3270: DeprecationWarning
```

Reply:
I think we can just get the 3.2 pre-release of Matplotlib installed on 3.8 instead - I'll push up a fix.